pdns/dnsdistdist/docs/reference/dnsnameset.rst

```diff
@@ -0,0 +1,60 @@
+.. _DNSNameSet:
+
+DNSNameSet objects
+==================
+
+A :class:`DNSNameSet` object is a set of :class:`DNSName` objects. 
+Based on std::set (usually implemented as red-black trees).
```

Comment:
This line is not true anymore; sorry about that.

Reply:
Fixed.